
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit4"/>
    </item>
    <item>
     <widget class="QPushButton" name="calcularButton">
      <property name="text">
       <string>Calcular Ganancia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultadoLabel">
      <property name="text">
       <string>Resultado:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
